src/geojson/geojson_intf.ml

```diff
@@ -216,12 +199,19 @@ module type S = sig
     end
   end
 
-  type t =
+  type geojson =
     | Feature of Feature.t
     | FeatureCollection of Feature.Collection.t
     | Geometry of Geometry.t  (** A geojson object *)
 
-  include Json_conv with type t := t and type json := json
+  type t = { geojson : geojson; bbox : float array option }
```

Comment:
What are your thoughts on exposing the implementation of this GeoJSON object type? Do you think we should make it abstract to the user i.e.

```ocaml
type t

val geojson : t -> geojson
val bbox : t -> float array option
```

Reply:
> What are your thoughts on exposing the implementation of this GeoJSON object type? Do you think we should make it abstract to the user i.e.
> 
> ```ocaml
> type t
> 
> val geojson : t -> geojson
> val bbox : t -> float array option
> ```

Abstracting as much as can be abstracted from the user while providing them with the maximum possible functionality is a good idea, I think.

However, I think we should still go with the current implementation. It's okay for the user to know how GeoJSON objects are represented.
If you feel otherwise though, there is no problem Sir.